app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+const Greetings = () => {
+  const [data, setData] = useState({}); // 1
+  const fetchData = async () => {
+    const result = await axios.get("http://localhost:3000/greetings");
+    setData(result.data);
+  };
+  console.log(data);
```

Comment:
- [ ] it will be great to remove the `console.log` statement. This should only be used in debugging and then removed from the code base. This is to ensure there are no leaks of sensitive information In your application.